chrome/global/browser.css

```diff
@@ -62,22 +62,22 @@ the menu bar. */
   }
 
   /* Disable rounded corners in fullscreen, only if the toolbar is hidden. */
-  :root[inFullscreen] #navigator-toolbox[style*="margin-top"] + #browser,
+  :root[inFullscreen] #navigator-toolbox[style*="margin-top"]+#browser,
   :root[inDOMFullscreen] #browser {
     --uc-tweak-rounded-corners-padding: 0;
     --uc-tweak-rounded-corners-radius: 0;
     --uc-tweak-rounded-corners-shadow: none;
   }
 
-  /* Apply rounded corners to the browser window. */
-  #appcontent {
+  /* Apply rounded corners to the browser container. */
+  html:not(html[inFullscreen="true"]) #tabbrowser-tabpanels>.browserSidebarContainer>.browserContainer {
```

Comment:
Is there any specific reason to change the #appcontent element to the .browserContainer element? It seems to cause issues with the sidebar and the colouring of the frame.

Also, the full screen part of that selector is also made redundant by the rule just above it.

Here is what it looks like with these changes:
![Latest commit pull-request screenshot](https://github.com/bmFtZQ/edge-frfox/assets/62812711/83fcbe52-b9f3-4f4c-9899-c97e4cda536c)

Here is what it looked like previously:
![Previous version screenshot](https://github.com/bmFtZQ/edge-frfox/assets/62812711/2ed0be7a-e202-45fd-a96d-36c674782f8e)

Reply:
Changing the #appcontent element to the .browserContainer element was intended to apply rounded corners only to the web content, such as separating the web content and DevTools similar to Microsoft Edge when using F12. However, it seems this change indeed broke some things, and I'm unsure how to fix it. Perhaps I should revert this part of the change?

Reply:
Based on some testing, these changes to the file seem to bring the same effect:

1. Replace this selector with just `#appcontent .browserContainer`.
2. Keep the `#browser` selector for the frame background colour.
3. Add a new rule to set the background colour of `#tabbrowser-tabpanels` to `transparent`.

After applying these changes it should work as intended, with the rounded corners being applied to the browser, but not the devtools.